README.md

```diff
@@ -66,6 +66,8 @@ The `extra.downloads` section contains a list of files to download. Each extra-f
 * `version`: (*Optional*) A version number for the downloaded artifact. This has no functional impact on the lifecycle of the artifact, but
    it can affect the console output, and it can be optionally used as a variable when setting `url` or `path`.
 
+* `variables_provider`: (*Optional*) A class name that implement `LastCall\DownloadsPlugin\Variables\VariablesProviderInterface`. Allow setting new variables with custom logic.
```

Comment:
If `variables_provider` is a terrible config name, or `\Variables\` is not good namespace, please give a better suggestion.

Reply:
Changed to `variables`